x/erc20/keeper/dynamic_precompiles.go

```diff
@@ -32,6 +40,29 @@ func (k Keeper) RegisterERC20Extension(ctx sdk.Context, denom string) (*types.To
 	return &pair, err
 }
 
+// RegisterERC20CodeHash sets the codehash for the erc20 precompile account
+// if the bytecode for the erc20 codehash does not exists, it stores it.
+func (k Keeper) RegisterERC20CodeHash(ctx sdk.Context, pair types.TokenPair) error {
+	contractAddr := common.HexToAddress(pair.Erc20Address)
+	var (
+		// bytecode and codeHash is the same for all IBC coins
+		// cause they're all using the same contract
+		bytecode = common.FromHex(types.Erc20Bytecode)
+		codeHash = crypto.Keccak256(bytecode)
+	)
+	// check if code was already stored
+	code := k.evmKeeper.GetCode(ctx, common.Hash(codeHash))
+	if len(code) == 0 {
+		k.evmKeeper.SetCode(ctx, codeHash, bytecode)
+	}
+
+	return k.evmKeeper.SetAccount(ctx, contractAddr, statedb.Account{
+		CodeHash: codeHash,
+		Nonce:    0,
+		Balance:  common.Big0,
+	})
+}
+
```

Comment:
should we also define an inverse function to this? I mean, updating the account code hash to be the emptyCodeHash. So we can use these two on the UpdateParams msg, when enabling/disabling a dynamic precompile